client/state/plugins/wporg/selectors.js

```diff
@@ -1,27 +1,39 @@
-const getPlugin = function( state, pluginSlug ) {
-	if ( ! state || ! state[ pluginSlug ] ) {
+export function getPlugin( state, pluginSlug ) {
+	let plugins;
+	// Some components just pass in the list of plugins, not the entire state.
```

Comment:
I agree, but it's not currently an easy fix. If you look at [the example I linked](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plugins/main.jsx#L87:L95), you'll see that we end up calling this selector inside a component, after fetching the plugins list from the flux PluginsStore. We don't have the plugin slugs until that point, and they're stored in component state rather than redux state or props — so we can't use the selector in the `mapStateToProps` function. The other option for this issue is to pass the entire state as a prop, but that seems like just as awkward a workaround.

This should be a short-lived workaround, since once the plugins flux store goes away, we can remove this.